### PR TITLE
[GOES TO FEATURE BRANCH] Main migration article tweaks

### PR DESCRIPTION
**Pull request by rhoboat**
This PR makes some small changes to the main article. The dedicated guides have been moved to the `infrastructure-live-multi-account-acme` repo.

Features:
- Example dependency graph for a dev account using RA v1 is shown
- LInks to each dedicated service migration guide. (https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/pull/50)
- A folder has been provided as well! It has all the setup files needed to prepare your `live` repo for migration. It will be released in [`infrastructure-live-multi-account-acme`](https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/releases).
- A video of a migration: https://drive.google.com/file/d/1ZKR-TH98nBdYox5IdjS4duXKcvrwZxCx/view?usp=sharing
    - If approved, figure out the right place for it. Would like to avoid bloating _this_ repo with a 300+ MB file. 
    - Youtube was suggested, with embedding here.

### CODE REVIEW

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [inline _posts/2021-05-04-how-to-update-your-ref-arch.adoc:450]: @yorinasub17 Would like your input on this! I moved the state file backup step earlier, to before they update the terragrunt.hcl, because I have seen on my system that after making the changes to the config, the state pull doesn't behave. I think it's because the source points to the service catalog, rather than the old infra-modules. Should this not be the case?

If we're not 100% on this, I would rather err conservatively and tell users to backup before any changes.

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [inline _posts/2021-05-04-how-to-update-your-ref-arch.adoc:483]: @yorinasub17 I have not personally checked whether it works to run `terragrunt state list` after having made changes to the terragrunt.hcl. Also note, previously, the state file backup have happened _after_ this step.

File: _posts/refarch2-guides/eks.adoc
Comment (rhoboat) [inline _posts/refarch2-guides/eks.adoc:31]: This is WIP because I wanted to just give a taste of it rather than put a lot of time into it.

Basically, it would describe how to migrate all 4 of the EKS modules in order, but also highlight the places where you need to run apply differently, or run some other things post-apply. This is the case for EKS cluster and EKS core services.

It's also the case that they will have to hardcode the OIDC arn/url the first go-around, and once everything's migrated, they can go back and use the dependency output of EKS cluster (the last thing they'll migrate) to replace those hardcodes. 

I can imagine it being confusing to go back and forth between the main article and these special-cases, so I would like a separate "main article" listing the high-level steps for just the EKS stuff.

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [reply _posts/2021-05-04-how-to-update-your-ref-arch.adoc:450]: @yorinasub17 Bump in case missed

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (yorinasub17) [inline _posts/2021-05-04-how-to-update-your-ref-arch.adoc:532]: These links are all broken when I render the website in docker. I think it has to do with how we setup the jekyll plugin but not entirely sure.

**Review by yorinasub17 — COMMENTED** [review]
LGTM other than the comments. I only loosely skimmed the contents of the dedicated guides because I already reviewed them in the old place.

I wanted to do a format check for asciidoc though, but I couldn't because of issues with the linking and I couldn't load the page. I also think there is a broken image asset link when I did `docker-compose up`:

```
web_1  | [2021-05-26 08:59:40] ERROR `/assets/img/guides/ref-arch-v2-upgrade/ref-arch-v2-dep-tree.png' not found.
```

I'm not familiar enough with the asciidoc jekyll setup to know what is wrong here...

**Comment by yorinasub17** [comment]
UPDATE: 4119374 fix broken image asset

File: _posts/refarch2-guides/eks.adoc
Comment (yorinasub17) [reply _posts/refarch2-guides/eks.adoc:31]: Yea that seems reasonable!

File: _posts/refarch2-guides/alb.adoc
Comment (rhoboat) [reply _posts/refarch2-guides/alb.adoc]: Done in https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/pull/50

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [reply _posts/2021-05-04-how-to-update-your-ref-arch.adoc]: Links updated to point to new location defined in https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/pull/50

File: _posts/refarch2-guides/eks.adoc
Comment (rhoboat) [reply _posts/refarch2-guides/eks.adoc]: @yorinasub17 Input on this? Will do in https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/pull/50

**Comment by rhoboat** [comment]
Thanks for the review, Yori, pinged you for another look. Dedicated guides moved to https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/pull/50.

File: _posts/refarch2-guides/eks.adoc
Comment (rhoboat) [reply _posts/refarch2-guides/eks.adoc]: Done, resolving!

**Comment by rhoboat** [comment]
@bwhaley Would like your review on this. Yori has pushed a commit so cannot approve.

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (yorinasub17) [inline _posts/2021-05-04-how-to-update-your-ref-arch.adoc:115]: Do you plan on updating this to the actual release tag eventually? Only reason I am asking is because this link currently is a 404.

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (yorinasub17) [inline _posts/2021-05-04-how-to-update-your-ref-arch.adoc:533]: Reminder to update this to release tag instead of `master` when we cut release.

**Review by yorinasub17 — APPROVED** [review]
LGTM other than nits about tags! (NOTE: will need approval from another person given my commit in here).

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [reply _posts/2021-05-04-how-to-update-your-ref-arch.adoc:115]: Yes. Please see https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/releases for the draft.

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [reply _posts/2021-05-04-how-to-update-your-ref-arch.adoc:533]: We can do this now if I can get feedback on https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/releases

**Comment by bwhaley** [comment]
Sorry I missed this before the approval, but it looks great to me.

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [reply _posts/2021-05-04-how-to-update-your-ref-arch.adoc:115]: Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/427

File: _posts/2021-05-04-how-to-update-your-ref-arch.adoc
Comment (rhoboat) [reply _posts/2021-05-04-how-to-update-your-ref-arch.adoc:533]: Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/427

**Pull request merged**
